src/matchers/toLog/predicate.js

```diff
@@ -0,0 +1,17 @@
+import { equals } from 'expect/build/jasmine_utils';
```

Comment:
I assume this path used to be correct, but it's now `jasmineUtils` instead of `jasmine_utils`.